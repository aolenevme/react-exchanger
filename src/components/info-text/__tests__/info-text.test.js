import React from "react";
import renderer from "react-test-renderer";

import InfoText from "../info-text.js";

describe("<InfoText />", () => {
    it("is rendered correctly", () => {
        expect(renderer
            .create(<InfoText>Bingo!</InfoText>)
            .toJSON()).toMatchSnapshot();
    });
});
