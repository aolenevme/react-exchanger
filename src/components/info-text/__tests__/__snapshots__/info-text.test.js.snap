// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoText /> is rendered correctly 1`] = `
.c0 {
  font-size: 0.8rem;
  color: #7199e6;
}

<span
  className="c0"
>
  Bingo!
</span>
`;
